x-pack/platform/plugins/shared/fleet/kibana.jsonc

```diff
@@ -31,6 +31,7 @@
       "fieldsMetadata",
       "logsDataAccess",
       "embeddable",
+      "savedSearch",
```

Comment:
🥳  all is approved ... just a qq, I'm curious : why was "savedSearch" added to several kibana.jsonc?

Reply:
Because we started using `toSavedSearchAttributes` here: https://github.com/elastic/kibana/pull/229897/commits/67a4f14ef14a5b9bc572d22d3d0292be2ccc4833

I might be missing something but since it is being used in that package, consumers of that package need to also ensure they are importing the `savedSearch` plugin (similar to `embeddable`).

Reply:
thx & congrats!